packages/platform-node/src/Undici.ts

```diff
@@ -2,6 +2,7 @@
  * @since 1.0.0
  */
 
+export { default } from "undici"
```

Comment:
How should I define `@since`?

Reply:
You can copy paste the jsdocs from the other export

Reply:
Done, although I'm not sure it's correct to say the default export has been available since 1.0.0?

Reply:
While platform is api unstable we are using 1.0.0 as the since tag for everything.